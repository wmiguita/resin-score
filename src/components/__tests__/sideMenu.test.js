import React from "react"
import renderer from "react-test-renderer"

import SideMenu from "../sideMenu"

describe( "SideMenu", () => {
  it( "should render SideMenu", () => {
    const sideMenu = renderer.create( <SideMenu /> ).toJSON()

    expect( sideMenu ).toMatchSnapshot()
  })
})
