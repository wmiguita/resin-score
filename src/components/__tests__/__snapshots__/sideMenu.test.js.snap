// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideMenu should render SideMenu 1`] = `null`;
